mod/blog/elgg-plugin.php

```diff
@@ -81,6 +81,7 @@
 	],
 	'widgets' => [
 		'blog' => [
+			'name' => elgg_echo('collection:object:blog'),
```

Comment:
why not add the standardized names to the languages files

Reply:
widget title/description are magics...

Reply:
probably your recent translation changes remove the language strings

Reply:
Yes

Reply:
i think this needs to be added to translation file
`widgets:{$id}:name`

Reply:
I don't see a point in having multiple keys translating to the same string.

Reply:
i do... as it allows people to influence translations more specifically... we have specific widget keys for multiple widgets at this moment, so it would be nice if this is done the same way in this PR.. we can discuss this 'duplication' some other time